guides/keras_cv/cut_mix_mix_up_and_rand_augment.py

```diff
@@ -0,0 +1,398 @@
+labels.  Please note that `keras_cv.layers.CutMix` and `keras_cv.layers.MixUp` expect
+`labels` to be one hot encoded.  This is because they modify the values held in `labels`
+in a way that is not possible with a sparse label representation.
+"""
+
+IMAGE_SIZE = (224, 224)
+BATCH_SIZE = 32
+num_classes = dataset_info.features["label"].num_classes
+
+
+def prepare(image, label):
+    image = tf.image.resize(image, IMAGE_SIZE)
+    image = tf.cast(image, tf.float32)
+    label = tf.one_hot(label, num_classes)
+    return {"images": image, "labels": label}
+
+
+def prepare_dataset(dataset, split):
+    if split == "train":
+        return (
+            dataset.map(lambda x, y: prepare(x, y), num_parallel_calls=AUTOTUNE)
+            .shuffle(10 * BATCH_SIZE)
+            .batch(BATCH_SIZE)
+        )
+    if split == "test":
+        return dataset.map(
+            lambda x, y: prepare(x, y), num_parallel_calls=AUTOTUNE
+        ).batch(BATCH_SIZE)
+
+
+def load_dataset(split="train"):
+    dataset = data[split]
+    return prepare_dataset(dataset, split)
+
+
+train_dataset = load_dataset()
+
+"""
+Let's inspect some samples from our dataset:
+"""
+
+
+def visualize_dataset(dataset, title):
+    plt.figure(figsize=(6, 6)).suptitle(title, fontsize=18)
+    for i, samples in enumerate(iter(dataset.take(9))):
+        images = samples["images"]
+        plt.subplot(3, 3, i + 1)
+        plt.imshow(images[0].numpy().astype("uint8"))
+        plt.axis("off")
+    plt.show()
+
+
+visualize_dataset(train_dataset, title="Before Augmentation")
+
+"""
+Great!  Now we can move onto the augmentation step.
+"""
+
+"""
+## RandAugment
+"""
+
+"""
+RandAugment performs a standard set of perturbations to an image based on the parameters
+`magnitude`, `magnitude_stddev`, `augmentations_per_image` and `rate`.  Each of these
+parameters should be tuned to fit your specific case.  You can read more about these
+parameters in the [RandAugment API documentation](/api/keras_cv/layers/rand_augment).
+
+To remedy this issue KerasCV offers RandAugment, a general purpose image data
+augmentation algorithm.  RandAugment has been shown to provide improved image
+classification results across numerous datasets, and only has two hyperparameters to
+configure.
+
+To use `RandAugment` you need to provide a few values:
+
+- `value_range` describes the range of values covered in your images
+- `magnitude` is a value between 0 and 10, describing the strength of the perturbations
+applied
+- `num_layers` is an integer telling the layer how many augmentations to apply to each
+individual image
+- (Optional) `magnitude_standard_deviation` allows `magnitude` to be randomly sampled
+from a distribution with a standard deviation of `magnitude_standard_deviation`
+- (Optional) `probability_to_apply` indicates the probability to apply the augmentation
+applied at each layer.
+
+Let's use KerasCV's RandAugment implementation.
```

Comment:
As per the original RandAugment paper, it's supposed to take the depth of the augmentation chain and strength per augmentation as its parameters. 

I understand the thoughtfulness behind introducing the additional parameters. Do we want to spend a few more sentences discussing their motivation and how they better enable the Keras users? 

Also, link to the original RandAugment layer would be great.

Reply:
Linked to paper!  I'm not convinced the average reader will actually be familiar with the original papers set of parameters and thus am not sure we should write about the differences.  thoughts?